kinesumer.go

```diff
@@ -638,6 +638,7 @@ func (k *Kinesumer) Refresh(streams []string) {
 }
 
 // Errors returns error channel.
+// you must read from the Errors() channel or Kinesumer will be deadlocked.
```

Comment:
It is not a deadlock situation technically. Error sending could be just hanged if there are no listeners. 

And, there is a way to fundamentally prevent hanging even if there are no error listeners.

```go
select {
case k.errors <- errors.WithStack(err)
default:
	// no error listeners
}
```

If there are no error listeners, `select` will execute the `default` statement, then we will be free from the hanging issues.

Reply:
Thank you for nice Insight!

In brief,
1. change all sending errors code to be able to ignore error if channel is full.
2. document about the change above in `Errors` function.